includes/class-wcs-exporter-cron.php

```diff
@@ -150,4 +148,35 @@ public static function delete_export_file( $file_name )  {
         }
     }
 
+	/**
+	 * Creates upload directory. Does nothing if directory and index/htaccess files already exist
+	 *
+	 * @since 2.1
+	 * @return void
+	 */
+	protected static function create_upload_directory() {
+		$base = self::$cron_dir;
+
+		if ( wp_mkdir_p( $base ) ) {
+			// create an empty index.php
+			if ( ! file_exists( $base . '/index.php' ) ) {
+				$file_handle = @fopen( $base . '/index.php', 'w' );
+
+				if ( $file_handle ) {
+					fwrite( $file_handle, '' );
+					fclose( $file_handle );
+				}
```

Comment:
Why the empty `fwrite()`? 

Do you think that a single command like `touch()` or `file_put_contents()` can be used to create an empty file than `fopen()`, `fwrite()` and `fclose().`